core/src/test/scala/org/broadinstitute/dsp/workbench/welder/BackgroundTaskSpec.scala

```diff
@@ -37,6 +37,13 @@ class BackgroundTaskSpec extends AnyFlatSpec with WelderTestSuite {
     res.toString shouldBe s"gs://testBucket/notebooks/test.Rmd"
   }
 
```

Comment:
can you add one more test `test.Rmddd`

Reply:
no...that's ok...it's nice to have it be green..but we haven't been very strict about it